### DISCUSSION TITLE
file as login

### DISCUSSION THREAD

**Discussion by CamelT0E** [dani-garcia/vaultwarden#3650] [Ideas]
I have to use a certificate and a password for a login. The certificate (file) serves as the username.
Would it be possible to set this in Vaultwarden?

Currently, I always have to download the file from the attachment first and delete it afterward. If there was a better way, it would be nice.

This is used in the German tax program [Elster](https://www.elster.de/eportal/login/softpse).

Thanks

**Comment by BlackDex** [comment]
Best would be to ask this at Bitwarden.
https://community.bitwarden.com/c/feature-requests/5/none
https://github.com/orgs/bitwarden/discussions/categories/password-manager

All the clients need to support this, and those are not maintained by this project.

**Reply by CamelT0E** [reply]
Thank you, I'm making a new post at Bitwarden
